### PR TITLE
Luke login routing

### PR DESCRIPTION
**Pull request by luke-kalen**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -41,10 +53,10 @@ function App() {
             <UserContextProvider>
               <PhotosContextProvider>
                 <SearchContextProvider>
-                <NavDrawer darkMode={darkMode} setDarkMode={setDarkMode} />
+                <NavDrawer logOut={logOut} />
```
Comment (brentonhershner) [inline src/components/App.js]: I thought you liked Dark Mode! Why are you pulling it out?

File: src/components/Home.js [hunk 2]
```diff
@@ -12,15 +13,16 @@ import FriendsList from './FriendsList';
...
   // Use context in this instance is for the friends list
   return (
     <div>
       <Typography variant="h2">
         HELLO WORLD!
         <br />
-        <ContextTest />
-        <FriendsList />
+        {/* <div>{`Hello ${value}`}</div> */}
+        {/* <FriendsList friends={friends} pending={pending} requested={requested}/> */}
```
Comment (brentonhershner) [inline src/components/Home.js]: Did you intend to remove the friend list?

**Review by brentonhershner — CHANGES_REQUESTED** [review]

File: src/components/App.js (hunk 1 above)
Comment (tonyjly) [reply src/components/App.js]: Please no. #saveDarkMode

**Review by tonyjly — APPROVED** [review]

**Review by brentonhershner — APPROVED** [review]

File: src/components/Home.js (hunk 2 above)
Comment (luke-kalen) [reply src/components/Home.js]: No I don't believe I did... I wonder if that was the result of fixing a merge conflict I had

**Pull request merged**